sway/commands/input/scroll_factor.c

```diff
@@ -0,0 +1,30 @@
+#include <stdlib.h>
+#include <string.h>
+#include "sway/config.h"
+#include "sway/commands.h"
+#include "sway/input/input-manager.h"
+
+struct cmd_results *input_cmd_scroll_factor(int argc, char **argv) {
+	struct cmd_results *error = NULL;
+	if ((error = checkarg(argc, "scroll_factor", EXPECTED_AT_LEAST, 1))) {
+		return error;
+	}
+	struct input_config *ic = config->handler_context.input_config;
+	if (!ic) {
+		return cmd_results_new(CMD_FAILURE,
+			"scroll_factor", "No input device defined.");
+	}
+
+	char *end;
```

Comment:
Can you set `errno = 0` right before calling `strtof`? Otherwise it might fail because of a previous `errno` value.